### ISSUE TITLE
[Safari] Some YouTube videos sometimes don't play

### ISSUE THREAD

**Issue by DAddYE** [uBlock-LLC/uBlock#918]
When I browse to youtube and there are ads within the video I cannot play it.

**Comment by gorhill** [comment]
- A URL where you confirm the issue occurs
- Your selection of filter lists
- Your custom filters
- Your dynamic filtering rules, if any
- Browser/version

**Comment by chrisaljoudi** [comment]
@DAddYE I've run a bunch of tests with µBlock for Safari on YouTube.com and have faced no issues. Please provide steps to reproduce — your selection of filter lists, custom filters, browser and µBlock version.

**Comment by DAddYE** [comment]
Hi sorry for the lack of details, sometimes it happens on any video sometimes (after I opened the issue) it doesn't.

Found right now one: https://www.youtube.com/watch?v=NJLrcG64ulU

I'm on Safari 8.0.3. Nothing in `My Filters` and nothing in `My Rules`.

I think I have just the default filters enabled here a screen of those active:

![screen shot 2015-03-04 at 4 13 30 pm](https://cloud.githubusercontent.com/assets/6537/6496520/bd12b142-c289-11e4-9853-fdf4b278aa9c.png)

**Comment by DAddYE** [comment]
To add more seems that when ublock is active it raises this issue (on the link provided)

```
[Error] InvalidStateError: DOM Exception 11: An attempt was made to use an object that is not, or is no longer, usable.
    setRequestHeader (html5player.js, line 229)
    (anonymous function) (html5player.js, line 229)
    forEach (html5player.js, line 50)
    send (html5player.js, line 229)
    vX (html5player.js, line 1034)
    uX (html5player.js, line 1032)
    o (html5player.js, line 1040)
    LX (html5player.js, line 1039)
    (anonymous function) (html5player.js, line 1039)
    (anonymous function) ([native code], line 0)
    eS (html5player.js, line 933)
    (anonymous function) (html5player.js, line 933)
    Pm (html5player.js, line 202)
    X (html5player.js, line 201)
    Sw (html5player.js, line 210)
    (anonymous function) ([native code], line 0)
```

**Comment by chrisaljoudi** [comment]
@DAddYE ah, got it. It looks like this was caused by the same thing as #878 — the fix is 83975124a41e135374f1555f241525b12b5b53a6 and will be in 0.8.9.3. It's not in 0.8.9.2, though.
